skyfield/iokit.py

```diff
@@ -350,9 +358,14 @@ def parse_leap_seconds(fileobj):
 def parse_celestrak_tle(fileobj):
     lines = iter(fileobj)
     for line in lines:
-        name = line.decode('ascii').strip()
-        line1 = next(lines).decode('ascii')
-        line2 = next(lines).decode('ascii')
+        if line.decode('ascii').strip()[0] == '0':
```

Comment:
I can't quite tell what the comparison to `'0'` is trying to accomplish — clicking through to one sample Celestrak file, I'm not seeing a leading `0` character on any of its lines:

https://celestrak.com/NORAD/elements/stations.txt

Reply:
Sorry, I'll look at the celestrak format specifically. The source that I was using for my elsets (space-track.org) always starts the name line with a "0", then a "1" for the first TLE line and a "2" for the second. I was under the impression that that was the standard.